lib/prelude.nim

```diff
@@ -17,7 +17,9 @@
 ##
 ## .. code-block:: nim
 ##   import os, strutils, times, parseutils, hashes, tables, sets, sequtils
-##   when not defined(js): import parseopt
+##   when not defined(js) or (NimMajor, NimMinor, NimPatch) >= (1, 5, 1):
```

Comment:
What's the point of this? It's not as if the stdlib is commonly imported by older Nim compilers. I would remove this when completely.

Reply:
It is correct for ``--useVersion`` but overkill. It can be removed but it can also stay.

Reply:
Removed, no real point given that it only adds support